app/scripts/controllers/transactions/index.js

```diff
@@ -542,6 +542,15 @@ class TransactionController extends EventEmitter {
     })
     this.pendingTxTracker.on('tx:failed', this.txStateManager.setTxStatusFailed.bind(this.txStateManager))
     this.pendingTxTracker.on('tx:confirmed', (txId) => this.confirmTransaction(txId))
+    this.pendingTxTracker.on('tx:dropped', (txId) => {
+      const txMeta = this.txStateManager.getTx(txId)
+      if (!txMeta.dropBlockBuffer) {
+        txMeta.dropBlockBuffer = 1
+        this.txStateManager.updateTx(txMeta, 'transactions/pending-tx-tracker#event: tx:drop-update')
+        return
+      }
+      this.setTxStatusDropped(txId)
```

Comment:
does this emit another tx dropped event?

Reply:
it emits a status change event